content/src/migrations/Helper.ts

```diff
@@ -0,0 +1,7 @@
+import { EntityId } from 'dcl-catalyst-commons'
+import { MigrationBuilder } from 'node-pg-migrate'
+
+export function deleteFailedDeployments(pgm: MigrationBuilder, ...entityIds: EntityId[]) {
```

Comment:
We are creating this helper function because we will probably need to repeat this process in the future